Library/Homebrew/formula_installer.rb

```diff
@@ -212,6 +212,11 @@ def prelude
   def verify_deps_exist
     begin
       compute_dependencies
+    rescue CoreTapFormulaUnavailableError => e
+      raise unless Homebrew::API::Bottle.available? e.name
+
+      Homebrew::API::Bottle.fetch_bottles(e.name)
```

Comment:
I don't particularly like `rescue` to handle expected behaviour in cases like this.